services/QuillLMS/engines/evidence/lib/evidence/evidence/profanity.rb

```diff
@@ -0,0 +1,26 @@
+module Evidence 
+  class Profanity 
+    def self.profane?(word)
+      return false unless word.respond_to?(:length) && word.length > 1
+      word = word.downcase.gsub(/[.!?]/, '')
+
+      BAD_WORDS.find do |badword|
+        stripped_badword = badword.gsub('*', '')
```

Comment:
There's enough logic in this loop, you might consider breaking it into its own method, i.e.
```ruby
BAD_WORDS.find {|bad_word| word_match?(bad_word, word)}